BackEndApp/authv1/emails.py

```diff
@@ -0,0 +1,28 @@
+from django.core.mail import send_mail
+from BackEndApp.settings import EMAIL_HOST_USER
+
+
+class Email_Templates:
+    def __init__(self, user, request):
+        self.user = user
+        self.request = request
+
+    def update_password(self, username, email, otp):
+        subject = "Auto-DL Request to Update Password"
+        msg = (
+            "You ("
+            + username
+            + ") requested to update your password. "
+            + "Your OTP is: "
+            + otp
+        )
+        send_mail(subject, msg, EMAIL_HOST_USER, [email])
```

Comment:
```suggestion
```

Please do not send email from this file. There's a reason it's called "EmailTemplates" right ? :)

Reply:
Made the necessary changes